package.json

```diff
@@ -7,21 +7,20 @@
   "author": "Bret Little <bret.little@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "prepublish": "npm run build",
+    "prepublish": "yarn build",
     "build": "babel src --out-dir lib --source-maps"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.5",
     "@babel/core": "^7.1.5",
     "@babel/preset-env": "^7.1.5",
-    "@babel/preset-react": "^7.0.0"
+    "@babel/preset-react": "^7.0.0",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0"
   },
   "dependencies": {
-    "disposable-component": "^0.3.1",
-    "react": "^16.6.3",
-    "react-disposable-decorator": "^4.0.0",
-    "react-dom": "^16.6.3",
-    "rx": "^4.1.0"
+    "disposable-component": "^0.3.2",
```

Comment:
See https://github.com/CanopyTax/disposable-component/releases/tag/v0.3.2 -- this is important for bundle sizes